std/math.d

```diff
@@ -4531,6 +4538,10 @@ public:
      /// A machine NaN was generated. (example: x = real.infinity * 0.0; )
      @property bool invalid() { return (flags & INVALID_MASK) != 0; }
 
+     /** A subnormal number was used in a calculation.
+     (example: x = real.min_normal / 2 + 1) */
+     @property bool subnormal() { return (flags & SUBNORMAL_MASK) != 0; }
```

Comment:
@andralex: Adding a new method here. As far as I remember, you want to review new names.